katdal/chunkstore_s3.py

```diff
@@ -52,6 +53,18 @@ def send(self, request, stream=False, timeout=None, *args, **kwargs):
         return super(_TimeoutHTTPAdapter, self).send(request, stream, timeout, *args, **kwargs)
 
 
+class _BearerAuth(requests.auth.AuthBase):
+    def __init__(self, token):
+        # Character set from RFC 6750
```

Comment:
Are we keeping OAuth 2.0 compatible tokens then?

Reply:
RFC 6750 isn't exclusive to OAuth 2.0. It can be used anywhere that a single token is used for authentication.